pkg/vpc/vpc.go

```diff
@@ -232,3 +239,21 @@ func ImportAllSubnets(provider api.ClusterProvider, spec *api.ClusterConfig) err
 
 	return nil
 }
+
+//UseEndpointAccessFromCluster retrieves the Cluster's endpoint access configuration via the SDK
+// as the CloudFormation Stack doesn't support that configuration currently
+func UseEndpointAccessFromCluster(provider api.ClusterProvider, spec *api.ClusterConfig) error {
+	input := &awseks.DescribeClusterInput{
+		Name: aws.String(spec.Metadata.Name),
```

Comment:
nit: can just use `Name: &spec.Metadata.Name`. I think main use case for `aws.String()` is if you have a string literal whose address you need.

Reply:
Changed.